app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/projectActionButton.tsx

```diff
@@ -7,60 +7,34 @@ import * as backendProvider from '../../providers/backend'
 import * as localBackend from '../localBackend'
 import * as modalProvider from '../../providers/modal'
 import * as svg from '../../components/svg'
+// Warning: This is a circular import.
```

Comment:
i'll move `ProjectActionButton` into `projectsTable.tsx`

as for the warning: it's not strictly needed, but i have a feeling that it would potentially cause problems if we ever stop bundling

Reply:
on second thought, it's too long to inline, i'll just extract `ProjectRowState` to its own file instead